src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -5987,6 +5987,68 @@ private ExpressionAst ErrorRecoveryParameterInExpression(ParameterToken paramTok
                     new CommandParameterAst(paramToken.Extent, paramToken.ParameterName, null, paramToken.Extent)});
         }
 
+        private ExpressionAst ListLiteralRule()
+        {
+            //G  list-literal-expression
+            //G      '['   new-lines:opt   list-entries   new-lines:opt  ']'
+            //G  list-entries
+            //G      unary-expression
+            //G      list-entries   ','   new-lines:opt   unary-expression
+
+            var lBracket = NextLBracket();
+            if (lBracket == null) { return null; }
+
+            var listValues = new List<ExpressionAst>();
+            Token commaToken = null;
+
+            do {
+                if (commaToken != null) { SkipToken(); }
+                V3SkipNewlines();
```

Comment:
It's probably time to get rid of `V3SkipNewlines` - I was overly cautious introducing that method and it hasn't been used.

Reply:
Will replace `V3SkipNewlines` with `SkipNewlines`